sdk/core/Azure.Core/tests/ClientDiagnosticsTests.Net50.cs

```diff
@@ -100,6 +100,7 @@ public void StartsActivitySourceActivity()
 
         [TestCase(null)]
         [TestCase(false)]
+        [TestCase(true)]
```

Comment:
What does `[TestCase(null)]` signify?

Reply:
The default value, that is suppression *off*, but can change after we decide to break old clients.